exla/examples/mnist.exs

```diff
@@ -1,11 +1,20 @@
+Mix.install([
+  {:nx, "~> 0.5"},
+  {:exla, "~> 0.5"}
+])
```

Comment:
Those examples are meant to run as `mix run examples/mnist.exs` from the EXLA project root. It is mostly for testing. Can you please remove the `Mix.install` from them and add such note? This means we won't be able to use Scholar though. :(

Reply:
Ok, I see